src/main/java/com/iridium/iridiumskyblock/listeners/PlayerJoinListener.java

```diff
@@ -29,7 +29,7 @@ public void onJoin(PlayerJoinEvent event) {
         if (player.isOp()) {
             Bukkit.getScheduler().runTaskLater(IridiumSkyblock.getInstance(), () ->
                             player.sendMessage(StringUtils.color(IridiumSkyblock.getInstance().getConfiguration().prefix + " &7Thanks for using IridiumSkyblock, if you like the plugin, consider donating at &bwww.patreon.com/Peaches_MLG"))
-                    , 5);
+                    , 10L);
```

Comment:
i just wanted adding L here but i think we should increase it to 2 seconds may it too early before player join and connection finished

Reply:
It's not